src/Raven.Server/Extensions/RawDatabaseRecordExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.ServerWide;
+using Raven.Server.ServerWide;
+using Sparrow.Json;
+
+namespace Raven.Server.Extensions;
+
+public static class RawDatabaseRecordExtensions
```

Comment:
why this isn't under the `RawDatabaseRecord`?

Reply:
It's needed only once